pages/spicedb/getting-started/first-steps.mdx

```diff
@@ -16,8 +16,8 @@ import { InlinePlayground } from '@/components/playground'
 
     <Cards>
         <Card arrow={true} title="Zanzibar" href="../concepts/zanzibar" />
-        <Card arrow={true} title="Schema" href="../concepts/schema" />
-        <Card arrow={true} title="Relationships" href="../concepts/relationships" />
+        <Card arrow={true} title="Writing a Schema" href="../concepts/schema" />
```

Comment:
I'm very tempted to change this link to be "developing a schema". Having a "first step" be "read the reference" is not very "first-steppy" to me

Reply:
That works for me 🤔